src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/EncoderManager.m

```diff
@@ -41,11 +41,8 @@ - (id)init;
 - (void)dealloc;
 
 - (void)reset:(id<MTLTexture>)destination
-           isDstOpaque:(jboolean)isDstOpaque
-    isDstPremultiplied:(jboolean)isDstPremultiplied
-                  isAA:(jboolean)isAA
-                  isText:(jboolean)isText
-                  isLCD:(jboolean)isLCD;
+         isAA:(jboolean)isAA
```

Comment:
I think by mistake isText argument is also removed. It is not matching reset() implementation.

Reply:
Thanks for the findings! Fixed.